snapcraft/cli/lifecycle.py

```diff
@@ -115,7 +115,9 @@ def init():
     """Initialize a snapcraft project."""
     snapcraft_yaml_path = lifecycle.init()
     echo.info("Created {}.".format(snapcraft_yaml_path))
-    echo.info("Edit the file to your liking or run `snapcraft` to get started")
+    echo.wrapped("Edit the file to your liking or run `snapcraft` to get started. "
+                 "For more information about the format and values accepted in "
+                 "snapcraft.yaml go to https://docs.snapcraft.io/the-snapcraft-format/8337.")
```

Comment:
That link sure looks permanent :upside_down_face: .

Also, note that no docs here (including https://docs.snapcraft.io/snapcraft-top-level-metadata/8334) include a description of `base`.

Reply:
but https://docs.snapcraft.io/the-snapcraft-format/8337 does, it is just a matter of adding the edit.

Reply:
and I just did